terms/PsychENCODESpecific/grant.json

```diff
@@ -0,0 +1,124 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id":"sysbio.PECSpecific-annotations.grant-0.0.1",
+    "description": "Grant number the data is being submitted for",
+    "type": "string",
+    "anyOf": [
+        {
+            "const": "R01MH109677",
```

Comment:
These will all need a source.

Reply:
If I remember correctly, I believe AD links to the grant as source. If PEC doesn't want to do that, you could just add `"source": ""` to all.